include/posix/pthread.h

```diff
@@ -19,6 +19,8 @@ struct timespec {
 };
 #endif
 
+typedef unsigned long useconds_t;
```

Comment:
According to POSIX (http://pubs.opengroup.org/onlinepubs/7908799/xsh/systypes.h.html), this is  defined in sys/types.h, why these are being put here? This *WILL* lead to header conflicts.

Reply:
Thanks for review, I will modify it. I was trying to enhance the existing POSIX implementation as started by @andyross, He has put the typedef into as part of pthread.h. Sure I got your point, I will take care of your comment in next patch version.